packages/components/loading-spinner/src/loading-spinner.tsx

```diff
@@ -27,6 +27,11 @@ const sizePerScale = {
 const positionOrigin = '20px';
 
 type TLoadingSpinnerProps = {
+  /**
+   * Set the amount of time to delay the loading spinner before it renders.
+   * The default value of 1000ms has been agreed upon by the design team for all usecases.
```

Comment:
This is meant for public documentation, no need to mention the agreement thing.

Reply:
noted, and updated.